chapters/ch35.md

```diff
@@ -4,32 +4,32 @@
 
 > You are working on speech recognition and have a dataset of audio clips. Suppose your dataset has many noisy audio clips so that even humans have 10% error. Suppose your system already achieves 8% error. Can you use any of the three techniques described in Chapter 33 to continue making rapid progress?
 
-Bạn đang làm về nhận dạng giọng nói và bạn có một tập dữ liệu là các đoạn âm thanh. Giả sử, tập dữ liệu của bạn có nhiều âm thanh nhiễu mà thậm chí con người còn mắc phải 10% lỗi. Giả sử, hệ thống của bạn đã đạt được 8% lỗi. Liệu bạn có thể sử dụng bất kỳ kỹ thuật nào trong ba kỹ thuật được mô tả trong Chương 33 để tiếp tục tiến bộ nhanh chóng?
+Bạn đang làm về nhận dạng giọng nói và có một tập dữ liệu là các đoạn âm thanh. Giả sử, tập dữ liệu đó có nhiều đoạn âm thanh nhiễu mà thậm chí con người cũng mắc phải 10% lỗi. Giả sử, hệ thống của bạn đã đạt được 8% lỗi. Liệu bạn có thể sử dụng bất kỳ kỹ thuật nào trong ba kỹ thuật được mô tả trong Chương 33 để tiếp tục tiến bộ nhanh chóng không?
 
 > If you can identify a subset of data in which humans significantly surpass your system, then you can still use those techniques to drive rapid progress. For example, suppose your system is much better than people at recognizing speech in noisy audio, but humans are still better at transcribing very rapidly spoken speech.
 
-Nếu bạn có thể xác định một tập dữ liệu con mà con người vượt qua được hệ thống của bạn một cách đáng kể, thì bạn vẫn có thể sử dụng các kỹ thuật đó để thúc đẩy tiến trình nhanh chóng. Ví dụ, giả sử hệ thống của bạn tốt hơn nhiều so với con người trong việc nhận dạng giọng nói trong âm thanh nhiễu, nhưng con người vẫn làm tốt hơn trong việc ghi lại lời nói rất nhanh.
+Nếu có thể xác định một tập dữ liệu con mà con người đạt chất lượng cao hơn đáng kể so với hệ thống của bạn, thì bạn vẫn có thể sử dụng các kỹ thuật đó để thúc đẩy tiến trình nhanh chóng. Ví dụ, giả sử hệ thống của bạn tốt hơn nhiều so với con người trong việc nhận dạng giọng nói trong âm thanh nhiễu, nhưng con người vẫn tốt hơn trong việc ghi lại lời nói rất nhanh.
 
 > For the subset of data with rapidly spoken speech:
 
 Đối với tập dữ liệu con với lời nói nhanh:
 
 > 1. You can still obtain transcripts from humans that are higher quality than your algorithm’s output.
 
-1. Bạn vẫn có thể lấy được bản ghi thoại từ con người với chất lượng cao hơn so với đầu ra thuật toán của bạn.
+1. Lấy bản ghi thoại từ con người với chất lượng cao hơn so với đầu ra thuật toán của bạn.
 
 > 2. You can draw on human intuition to understand why they correctly heard a rapidly spoken utterance when your system didn’t.
 
-1. Bạn có thể dựa vào trực giác để hiểu lý do tại sao họ nghe chính xác một phát ngôn nhanh khi hệ thống của bạn chưa thể.
+2. Dựa vào trực giác để hiểu lý do tại sao họ nghe chính xác một phát ngôn nhanh khi hệ thống của bạn chưa thể.
```

Comment:
```suggestion
2. dựa vào trực giác để hiểu lý do tại sao họ nghe chính xác một phát ngôn nhanh khi hệ thống của bạn chưa thể.
```

Reply:
ủa tại sao chỗ này lại không viết hoa nhỉ?

Reply:
Vì cậu định không viết chủ ngữ cho các gạch đầu dòng này nên tớ để chủ ngữ lên trước dấu hai chấm. Việc này được hiểu là cả ba gạch đầu dòng này vẫn thuộc câu trước dấu hai chấm như đã gợi ý ở https://github.com/aivivn/Machine-Learning-Yearning-Vietnamese-Translation/pull/443#discussion_r356983878

Reply:
tớ vẫn thấy nên viết hoa đầu dòng. Vì khi gạch đầu dòng mình thường viết hoa hơn là viết thường.

Reply:
Mời xem lại [ở đây](https://ruthlesseditor.com/how-to-create-and-punctuate-bullet-points/) nha.

## Bullet punctuation choices
### Capitalize the first word and end with a period when the bullet point is a complete sentence.

Follow these USPS guidelines when you address your envelope:

* Write the name and full address in capital letters.
* Use the standard two-letter state abbreviation.
* Keep the ZIP code on the same line as the city and state.

### Lowercase the first word when the bullet point is not a complete sentence, and don’t add a period.

USPS address format guidelines:

* all capital letters
* two-letter state abbreviation
* city, state, ZIP code+4 on same line

Reply:
À không, ý mình là trong tiếng Việt cơ. Tiếng Anh thì cái này mình biết. Chắc chưa đọc văn bản nào có cái vụ này. 

Anyway, mình sửa theo cũng được.

Reply:
Bạn dẫn nguồn một tài liệu tiếng Việt nói về vấn đề này được không? Mình thấy tiếng Việt ít nói về quy chuẩn trong văn viết, đâm ra mỗi ông viết một kiểu. Nếu không có chuẩn nào rõ ràng thì mình cứ theo tiếng Anh.

Reply:
Ý là mình cũng chưa thấy trường hợp viết như này. Tìm cũng không biết tìm quy chuẩn thế nào nên mình cũng approve sửa như vậy đó.